PARCEL_BRISK_README.md

```diff
@@ -0,0 +1,12 @@
+# Contributing to Parcel/Brisk integration
+
+1. After cloning this repo, run `yarn && bolt`.
```

Comment:
Is this preferred over `yarn && yarn bolt`? Using a global bolt (which has an unpredictable version) over a local one?

Reply:
The `package.json` of brisk specifies some bolt config:

```js
  "bolt": {
    "version": "^0.23.2",
    "workspaces": [
      "packages/*"
    ]
  },
```

With bolt, it will error if your global install isn't compatible with what the repository wants.

Bolt is pretty greedy and just wants to own everything about installing. Pretty sure the first thing it does is run yarn, but bolt uses its local yarn for actions, so you could get errors where your yarn and bolt's yarn aren't the same.

Mostly ¯\_(ツ)_/¯ especially since we may end up moving to yarn workspaces.